D3D12Backend/Camera.cpp

```diff
@@ -70,7 +70,8 @@ namespace Boolka
     void Camera::UpdateInput(float deltaTime, const Vector4& right, const Vector4& up,
                              const Vector4& forward)
     {
-        static const float defaultMoveSpeed = 15.0f;
+        //TODO move into separate confing maybe?
+        static const float defaultMoveSpeed = 35.0f;
```

Comment:
Please don't change default params in master. It's meant for default scene.
```suggestion
        static const float defaultMoveSpeed = 15.0f;
```